vispy/visuals/image.py

```diff
@@ -276,7 +269,7 @@ def __init__(self, data=None, method='auto', grid=(1, 1),
         # self._build_interpolation()
         self._data_lookup_fn = None
 
-        self.clim = clim
+        self.clim = clim or "auto"  # None -> "auto"
```

Comment:
The default in the `__init__` definition is `'auto'` so if this is provided as `None` then it must have been set explicitly by the user. The docstring does not say it can be `None`. I'd say either update the docstring or undo this change. I don't feel strongly about it.

Reply:
I like the idea of "None means default" where it is applicable. I updated the docs of both Image and Volume to align them.

Reply:
I agree. Thanks.